LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIOutpostManagement.uc

```diff
@@ -83,176 +94,347 @@ function OnRadioTowerUpgradeClicked(UIButton Button)
 
 simulated function InitScreen(XComPlayerController InitController, UIMovie InitMovie, optional name InitName)
 {
-    local int NextY;
-    local XComGameState_WorldRegion Region;
-    local XComGameState_LWOutpost Outpost;
-    local XComGameStateHistory History;
-	local string strRegionalInfo;
-    local string strResistanceMecs;
-	local string strJobDetail;
-	local XGParamTag kTag;
-	local XComGameState_WorldRegion_LWStrategyAI RegionalAIState;
 	local bool IntelProhibited, SupplyProhibited, RecruitProhibited;
-	local int k;
+	local int AvailableSpace, i, AdviserIconSize, AdviserBorderPadding, ScrollbarPadding;
 
- 	// Init UI
-	super.InitScreen(InitController, InitMovie, InitName);
+    local int NextX, NextY;
```

Comment:
Indent is a mess in the existing Long War 2 code base. It should be tabs for indent rather than spaces.